hugo/content/en/examples/shortcodes/quote/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Quote
+weight: 28
+---
+
+Next to the regular (very basic) [markdown blockquotes](/examples/basic/block#blockquotes), there is an option to use the `{{</* quote */>}}` shortcode. With this shortcode you can also add an author name and their role. But unlike the markdown blockquotes, the `quote` shortcode does not accept markdown.
```

Comment:
Maybe also link to this page from the basic/block page quote section?  
"For advanced quote shortcode see: .."